src/fmu/sumo/sim2sumo/_special_treatments.py

```diff
@@ -193,6 +220,54 @@ def vfp_to_arrow_dict(datafile, options):
     return keyword, arrow_tables
 
 
+def add_md_to_rft(rft_table, md_file_path):
+    """Merge md data with rft table
+
+    Args:
+        rft_table (pd.DataFrame): the rft dataframe
+        md_file_path (str): path to file with md data
+
+    Raises:
+        FileNotFoundError: if md_file_path does not point to existing file
+
+    Returns:
+        pd.Dataframe: the merged results
+    """
+    logger = logging.getLogger(__file__ + ".add_md_to_rft")
+    logger.debug("Head of rft table prior to merge:\n %s", rft_table.head())
+
+    try:
+        md_table = pd.read_csv(md_file_path)
+    except FileNotFoundError as fnfe:
+        raise FileNotFoundError(
+            f"There is no md file called {md_file_path}"
+        ) from fnfe
+
+    xtgeo_index_names = ["I_INDEX", "J_INDEX", "K_INDEX"]
+    rft_index_names = ["CONIPOS", "CONJPOS", "CONKPOS"]
+    md_table[
+        xtgeo_index_names
+    ] += 1  # for grid indeces xtgeo starts from 0, res2df from 1
```

Comment:
The comment makes this formatting weird
Can we make it: `md_table[xtgeo_index_names] += 1` to increase readability?

Reply:
Not an important thing to fix. Just me being nitpicky cause I'm not that fluent in pandas 😅

Reply:
Yes, maybe a bit on the detailed side, but fixed anyhow